elyra/metadata/schemas/kfp.json

```diff
@@ -77,7 +77,8 @@
           "default": "Argo",
           "uihints": {
             "field_type": "dropdown",
-            "category": "Kubeflow Pipelines"
+            "category": "Kubeflow Pipelines",
+            "placeholder": "Select engine type"
```

Comment:
But I believe this is one of those places that most people will be using the default engine that is deployed with KFP which is Argo. Should we actually have a `default value` or Argo here? Which is what we do if this is not selected?

Reply:
See https://github.com/elyra-ai/elyra/pull/1443#issuecomment-801223295, which proposes a simpler solution that eliminates the need for this placeholder altogether for this specific property.